src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/TestHelpers.cs

```diff
@@ -16,13 +15,23 @@ public static class TestHelpers
 
         private static string s_tcpUrl;
         private static string s_httpUrl;
+        private static string s_tcpDupUrl;
+        private static string s_httpDupUrl;
+        private static string s_tcpStreamingUrl;
+        private static string s_httpStreamingUrl;
 
+       
         public static void SetHostAndProtocol(bool useHttp, string hostName, string appName)
         {
             UseHttp = useHttp;
             HostName = hostName;
             s_tcpUrl = "net.tcp://" + hostName + ":808/" + appName + "/Service1.svc";
             s_httpUrl = "http://" + hostName + "/" + appName + "/Service1.svc";
+            s_tcpDupUrl = "net.tcp://" + hostName + ":808/" + appName + "/DuplexService.svc";
+            s_httpDupUrl = "http://" + hostName + "/" + appName + "/DuplexService.svc";
+            s_tcpStreamingUrl = "net.tcp://" + hostName + ":808/" + appName + "/StreamingService.svc";
+            s_httpStreamingUrl = "http://" + hostName + "/" + appName + "/StreamingService.svc";
+            //s_httpUrl = HttpUrl;
```

Comment:
For future -- these are ideal for using TestProperties.  Can we remove the commented out //s_httpUrl?  Not required for merge.